Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.ts

```diff
@@ -65,18 +67,18 @@ export class CustomGroupComponent implements OnInit, OnDestroy {
     this.editMode = !!this.groupName;
 
     if (this.editMode) {
-      Observable.zip(this.getGroupRoles(), this.getGroupUsers())
-        .do((result: [IRole[], IUser[]]) => {
+      Observable.zip(this.getGroupRoles(), this.getGroupUsers(), this.getChildGroups())
+        .do((result: [IRole[], IUser[], IGroup[]]) => {
           this.roles = result[0];
-          this.associatedPrincipals = result[1];
+          this.associatedUsers = result[1];
```

Comment:
Similar comment as above, could we combine the results from the two calls into on associatedPrincipals array?